test.py

```
@@ -1,13 +1,7 @@
 from azure.identity import DefaultAzureCredential
```

Ruff (0.8.2)

`azure.identity.DefaultAzureCredential` imported but unused

Remove unused import: `azure.identity.DefaultAzureCredential`

(F401)